src/main/java/com/redhat/rhjmc/containerjfr/net/web/handlers/TargetRecordingsPostHandler.java

```diff
@@ -162,6 +162,22 @@ void handleAuthenticated(RoutingContext ctx) throws Exception {
                                                     TimeUnit.SECONDS.toMillis(
                                                             Long.parseLong(attrs.get("duration"))));
                                 }
+                                if (attrs.contains("toDisk")) {
+                                    Pattern bool = Pattern.compile("true|false");
+                                    Matcher m = bool.matcher(attrs.get("toDisk"));
+                                    if (!m.find())
+                                        throw new NumberFormatException(
```

Comment:
The other handler looks like it directly throws an `HttpStatusException`, but this one is still a `NumberFormatException` - they should probably be consistent with each other

Reply:
I had issues throwing HttpStatusExceptions inside executeConnectedTask, so I changed the PATCH handler to match this one.

Reply:
What kind of issues?

I think think the `NumberFormatException` is out of place when parsing a boolean.

Reply:
Ok, I went back to the old format, and throw a HttpStatusException for the boolean instead of the NumberFormatException